transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__all.sql

```diff
@@ -0,0 +1,255 @@
+
+with source as (
```

Comment:
The name of the model is too generic. Perhaps it can be renamed to `stg_mm_telemetry_prod__configs` or `stg_mm_telemetry_prod__consolidated_configs`.